### DISCUSSION TITLE
Share Your Best Resources!

### DISCUSSION THREAD

**Discussion by hesamsheikh** [hesamsheikh/ml-retreat#1] [General]
![image](https://github.com/user-attachments/assets/5ac325dc-6484-41f9-b3da-edeefe95e16f)

### Hey ML geeks, 

This is my first time starting a discussion so I'm pretty noob in this stuff 😅. 
I thought the biggest thing this repository needs is a place to share. I know the ML community is pretty cracked when it comes to finding better material for a subject. There is so much out there to learn that there is no **"best"** resources. 

I share what I find, on my study days. But many of you may be more familiar with some dope material, **so I made this discussion as a place to share.**

You can share any interesting books, videos, articles, or even tweets if you may. As long as it's related to machine learning or specifically about the topics I have covered, as long as it's fun, keep it coming :)

**Comment by akylzx** [comment]
yo
